static/linux/share/applications/webtorrent-desktop.desktop

```diff
@@ -0,0 +1,33 @@
+[Desktop Entry]
```

Comment:
Is there any way to unify this with `static/linux/webtorrent-desktop.desktop`? Looks mostly identical.